File: crates/fresh-editor/tests/fixtures/syntax_highlighting/hello.ex
# Elixir syntax highlighting test
defmodule Hello do
  @moduledoc "A greeting module"

  def greet(name) do
    "Hello, #{name}!"
  end

  def main do
    message = greet("World")
    IO.puts(message)

    Enum.each(1..5, fn i ->
      IO.puts("Item: #{i}")
    end)
  end
end

File: crates/fresh-editor/tests/fixtures/syntax_highlighting/hello.erl
-module(hello).
-export([greet/1, main/0]).

%% Greeting function
greet(Name) ->
    io_lib:format("Hello, ~s!", [Name]).

main() ->
    Message = greet("World"),
    io:format("~s~n", [Message]),
    lists:foreach(fun(I) ->
        io:format("Item: ~p~n", [I])
    end, [1, 2, 3]).
